extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -158,8 +165,83 @@ export class AssessmentDetailsHeader {
 		}).component();
 
 		cardContainer.addItems([cardHeading, cardText]);
-		this._valueContainers.push(cardText);
 
+		if (linkMetaData.title === constants.RECOMMENDED_CONFIGURATION) {
+			this._linksContainer = this._view.modelBuilder.flexContainer().withProps({
+				CSSStyles: {
+					'display': 'flex',
+					'flex-direction': 'row',
+					'width': '190px'
+				}
+			}).component();
+
+
+			this._viewDetailsLink = this._view.modelBuilder.hyperlink().withProps({
+				label: constants.VIEW_DETAILS,
+				url: '',
+				height: 18,
+				CSSStyles: {
+					'font-size': '13px',
+					'font-weight': '400',
+					'line-height': '18px',
+					'text-decoration': 'none',
```

Comment:
same. extract it out in styles.ts and use it at all places.

Reply:
done